setup.py

```diff
@@ -5,12 +5,14 @@
 
 setup(
     name="umami",
-    python_requires=">=3.6",
+    python_requires=">=3",
```

Comment:
@mcflugen I'm pretty certain this is the only change that needs to be made (+ a release). Right?